main.py

```diff
@@ -21,26 +22,32 @@
 
 # ================================
 
-class EnableStatus(ndb.Model):
+class EnableStatus(ndb.Model): #NDB entity called EnabledStatus
     # key name: str(chat_id)
-    enabled = ndb.BooleanProperty(indexed=False, default=False)
+    enabled = ndb.BooleanProperty(indexed=False, default=False) #Entity has atribute enabled
 
 
 # ================================
 
 def setEnabled(chat_id, yes):
-    es = EnableStatus.get_or_insert(str(chat_id))
-    es.enabled = yes
+    es = ndb.Key(EnableStatus, str(chat_id)).get() #Gets the entity
```

Comment:
do not change this. Please keep it as es = EnableStatus.get_or_insert(str(chat_id))